hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -186,20 +186,25 @@
     }
     {
       name: sw_test_mode_common
-      run_opts: ["+sw_build_bin_dir={sw_build_dir}/build-bin",
+      run_opts: ["+sw_build_bin_dir={run_dir}",
```

Comment:
This can be removed - SV (i.e. the simulator) does not need to know the path if it is `cwd`.

Reply:
Fixed.